code/modules/events/ion_storm.dm

```diff
@@ -19,18 +20,26 @@
 	if(announceEvent == ION_ANNOUNCE || (announceEvent == ION_RANDOM && prob(ionAnnounceChance)))
 		GLOB.event_announcement.Announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/AI/ionstorm.ogg')
 
-
 /datum/event/ion_storm/start()
 	//AI laws
 	for(var/mob/living/silicon/ai/M in GLOB.living_mob_list)
-		if(M.stat != 2 && M.see_in_dark != 0)
+		if(M.stat != DEAD && M.see_in_dark != FALSE)
```

Comment:
What kind of check is this. see_in_dark? wot
Is this to check if they are carded or....
Nothing to do with your PR but just something I noticed

Reply:
:man_shrugging: